source/includes/Encounter/encounter.md

```diff
@@ -233,16 +230,16 @@ POST encounter/:target_encounter_uuid/encounterprovider/:target_encounter_provid
   "encounterRole": "240b26f9-dd88-4172-823d-4a8bfeb7841f"
 }
 ```
-## Delete encounter provider sub resource
+## Delete encounter provider subresource
 
-* Delete or Voided a target encounter provider sub resource by its UUID. Returns a `404 Not Found` status if attribute not exists. 
- If user not logged in to perform this action, a `401 Unauthorized` status returned.
+* Delete or Voided a target encounter provider subresource by its UUID. Returns a `404 Not Found` status if attribute not exists. 
```

Comment:
```suggestion
* Delete or Voided a target encounter provider sub-resource by its UUID. Returns a `404 Not Found` status if attribute not exists. 
```

Reply:
@sanjulamadurapperuma In the docs in other sections as well subresource is used uniformly all over thats why I have adhered to it here as well.

Reply:
@LuGO0 It should actually be _sub-resources_ in any context. We have used _sub-resources_ in our [Wiki](https://wiki.openmrs.org/display/docs/Documenting+REST+Resources) as well. Could you point to some other doc sections using _subresources_?